tests/test_plugin_features.py

```diff
@@ -0,0 +1,136 @@
+import pathlib
+import numpy as np
+import pytest
+
+import dclab
+from dclab.rtdc_dataset.plugins.plugin_feature import (
+    PlugInFeature, find_plugin_feature_script, remove_all_plugin_features)
+
+from helper_methods import retrieve_data
+
+
+@pytest.fixture(autouse=True)
+def cleanup_plugin_features():
+    """Fixture used to setup and cleanup some fake ancillary features"""
+    # code run before the test
+    pass
+    # then the test is run
+    yield
+    # code run after the test
+    # remove our test plugin examples
+    remove_all_plugin_features()
+
+
+def get_plugin_file(plugin_name="plugin_test_example.py"):
+    plugin_path = pathlib.Path(__file__).parent / "data" / plugin_name
+    return plugin_path
+
+
+def test_create_plugin():
+    plugin_path = get_plugin_file()
+    plugin_list = dclab.load_plugin_feature(plugin_path)
+    assert isinstance(plugin_list[0], PlugInFeature)
+    assert isinstance(plugin_list[1], PlugInFeature)
+
```

Comment:
another critical assertion is `assert dclab.dfn.feature_exists("circ_per_area")`.